content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/150-referential-actions.mdx

```diff
@@ -44,15 +46,15 @@ You can also decide to skip checking the defaults and run a migration to update
 
 ### Using Introspection
 
-If you [Introspect](../../../../concepts/components/introspection) your database, the referential actions configured at the database level will be reflected in your Prisma Schema. If you have been using Prisma Migrate or `prisma db push` to manage the database schema, these are likely to be the [<=2.25.0 default values](#the-2250-default-referential-actions).
+If you [Introspect](/concepts/components/introspection) your database, the referential actions configured at the database level will be reflected in your Prisma Schema. If you have been using Prisma Migrate or `prisma db push` to manage the database schema, these are likely to be the <=2.25.0 default values.
```

Comment:
last link got lost. intentional?

Reply:
no that was intentional. We no longer have an anchor to link to. _But_ thinking about it, this makes no sense for the reader, they have no context here. So i will add an a element with an id and link to that. The default values do exist in the top part of the ref actions concepts page